src/lib/editable-element.ts

```diff
@@ -122,10 +123,23 @@ export class EditableElement {
 
 	private toggleItemHover(hover: boolean, event: MouseEvent) {
 		if (this.element !== event.currentTarget || this.hover === hover) return;
+
 		this.hover = hover;
+		this.setParentsHover();
 		this.overlayElement.toggleHover(hover);
 	}
 
+	private setParentsHover() {
+		const hoveredItems = EditableStore.getHoveredItems();
+
+		hoveredItems.forEach((element) => {
```

Comment:
```suggestion
		hoveredItems.forEach((hoveredItem) => {
```

I feel like we should maybe change this so as to avoid `element.element` below which is a bit unclear